### PR TITLE
Fix Doctrine ORM deprecations

### PR DESCRIPTION
**Pull request by dmitryuk**

### CODE REVIEW

File: src/Provider/Doctrine/Persistence/Helper/DoctrineHelper.php [hunk 1]
```diff
@@ -55,4 +67,92 @@ public static function getDoctrineType(string $type): string
...
+
+            return ORMSetup::createAnnotationMetadataConfiguration($paths, $isDevMode);
+        }
+
+        return Setup::createAnnotationMetadataConfiguration($paths, $isDevMode, null, null, false);
+    }
+
+    public static function getEntityManagerFromOnFlushEventArgs(OnFlushEventArgs $args): EntityManagerInterface
+    {
+        return method_exists($args, '') ? $args->getObjectManager() : $args->getEntityManager();
```
Comment (andrew-demb) [inline src/Provider/Doctrine/Persistence/Helper/DoctrineHelper.php:156]:
```suggestion
        return method_exists($args, 'getObjectManager') ? $args->getObjectManager() : $args->getEntityManager();
```

Comment (dmitryuk) [reply src/Provider/Doctrine/Persistence/Helper/DoctrineHelper.php]: Really missed it. Done

**Comment by dmitryuk** [comment]
@DamienHarper ready for review

File: src/Provider/Doctrine/Persistence/Helper/DoctrineHelper.php [hunk 2]
```diff
@@ -4,13 +4,26 @@
...
+use Doctrine\ORM\Configuration;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Event\OnFlushEventArgs;
+use Doctrine\ORM\ORMSetup;
+use Doctrine\ORM\Tools\Setup;
 use InvalidArgumentException;
 
 /**
  * @see \DH\Auditor\Tests\Provider\Doctrine\Persistence\Helper\DoctrineHelperTest
+ * @internal
```
Comment (dmitryuk) [inline src/Provider/Doctrine/Persistence/Helper/DoctrineHelper.php:24]: Nobody outside should use the file

**Comment by dmitryuk** [comment]
@DamienHarper what do you think? I need this MR in https://github.com/DamienHarper/auditor/pull/123

**Pull request merged**